_locales/en/messages.json

```diff
@@ -458,18 +458,31 @@
         "message": "EEPROM <span style=\"color: #37a8db\">saved</span>"
     },
 
-    "featureRX_PPM": {
+    "RX_PPM": {
         "message": "PPM RX input"
     },
+    "RX_SERIAL": {
+        "message": "Serial-based receiver (SPEKSAT, SBUS, SUMD)"
+    },
+    "RX_PARALLEL_PWM": {
+        "message": "PWM RX input (one wire per channel)"
+    },
+    "RX_MSP": {
+        "message": "MSP RX input (control via MSP port)"
+    },
+    "RX_NRF24": {
```

Comment:
Shouldn't we rename `RX_NRF24` to `RX_SPI` for consistency?

Reply:
We can take the opportunity to do so. On which order should they appear in the combobox? I guess `RX_SERIAL` should be the first one, not sure about the rest though.

Reply:
Let's make it SERIAL, PPM, PWM, SPI, MSP, NONE